packages/dds/map/src/interfaces.ts

```diff
@@ -334,12 +334,17 @@ export interface ISharedMapEvents extends ISharedObjectEvents {
 }
 
 /**
- * The SharedMap distributed data structure can be used to store key-value pairs. It provides the same API for setting
- * and retrieving values that JavaScript developers are accustomed to with the
+ * The SharedMap distributed data structure can be used to store key-value pairs.
+ *
+ * @remarks
+ * SharedMap provides the same API for setting and retrieving values that JavaScript developers are accustomed to with the
  * {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map | Map} built-in object.
  * However, the keys of a SharedMap must be strings, and the values must either be a JSON-serializable object or a
  * {@link @fluidframework/datastore#FluidObjectHandle}.
  *
+ * Note: unlike JavaScript maps, SharedMap does not make any guarantees regarding enumeration order.
+ * If your scenario requires consistent key ordering, you will need to manage that ordering yourself.
```

Comment:
nit: it's probably not necessary to include this sentence, given how strong the above sentence is, and it's also not clear what it means (ordering myself, like sort? or is there some other way to order the keys of the sharedtree?). The docs on the iterators themselves give a suggestion ("sort the entries")

Reply:
Fair. I'll just delete the second sentence then.
```suggestion
```